views/beneficial-owner-other.html

```diff
@@ -12,9 +12,10 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-
-      <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the corporate beneficial owner</h1>
-      <p class="govuk-caption-l">You can add more corporate beneficial owners later</p>
+      <span class="govuk-caption-xl">Beneficial owner</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the other legal entity</h1>
+      <br>
+      <p class="govuk-body">You can add more later.</p>
       <br>
       <form class="form" method="post">
 
```

Comment:
This summary text should be removed

Reply:
Removed! Really?

Reply:
On the ticket:
2. Tell us about the corporate beneficial owner’ changed to ‘Tell us about the other legal entity’
3. Text ‘You can add more later.’ has been added

Unless you mean something else.

Reply:
The summary text is on the line below

Reply:
You have removed it, great